params/config.libevm.go

```diff
@@ -181,8 +199,10 @@ func (e ExtraPayloads[C, R]) hooksFromChainConfig(c *ChainConfig) ChainConfigHoo
 }
 
 // FromRules returns the Rules' extra payload.
-func (ExtraPayloads[C, R]) FromRules(r *Rules) R {
-	return pseudo.MustNewValue[R](r.extraPayload()).Get()
+//
+// Deprecated: use the equivalent [ExtraPayloads.Rules] method.
```

Comment:
nit: IMO we can delete the "Deprecated" ones, as we don't need legacy code accumulating in libevm.

Reply:
I'm very happy to remove them and will do so before merging. I left them in so you wouldn't have to modify your code.